### ISSUE TITLE
Summarise updated issues?

### ISSUE THREAD

**Issue by originalfoo** [DoESLiverpool/weeknote-generator#21]
Currently there's no metric on how many issues have been updated in the past week (only new + closed). Could we add a metric in the issue summary line, for example:

> Summary: 170 open (N modified), 316 closed

Where N is number updated within past week. Should just be a case of including the `updated` search term in to query used for new tickets.

Ruby kinda melts my brain otherwise I'd submit a PR for this.

**Comment by amcewen** [comment]
Good idea, and as I was in the repo tweaking a few other things just now, I've added this.  It'll show up like this:

> Issue counts: 415 open; 1702 closed; 6 updated this week
